needrestart

```diff
@@ -92,6 +92,7 @@ Usage:
     -b		enable batch mode
     -p          enable nagios plugin mode
     -f <fe>	override debconf frontend (DEBIAN_FRONTEND, debconf(7))
+    -t <number> tolerate process start times within this value
```

Comment:
Missing time unit, same as above. Maybe replace process by script (it tolerates the timestamp difference between 
a process running from a scripting/VM langauge).

Reply:
I changed "number" to "seconds".

I changed "process" to "script process" to be most precise. Is that ok?

Reply:
I wonder if it might be better to add the term [interpreter](https://github.com/liske/needrestart#interpreters) to the `needrestart(1)` and `--help` since this is the name of this feature in other parts of the documentation/README.

Sorry for holding back this PR again.

Reply:
You mean instead of "script"? That makes sense and is consistent.

Done; I changed the surrounding wording a bit to make the change fit well in context.

Reply:
:+1: